style.css

```diff
@@ -1,88 +1,295 @@
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  margin-bottom: 20px;
+  margin-top: 12px;
+}
+
+.works .li-elements {
   display: flex;
+  list-style: none;
+  padding-left: 0;
 }
 
-.social-media .headline {
-  color: #7f8cff;
-  font-size: 16px;
+.li-elements li {
+  margin-right: 12px;
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+ul .current-li {
+  color: #344563;
+}
+
+.works p {
+  font-family: 'poppins', Arial, Helvetica, sans-serif;
+  font-weight: 400px;
+  font-size: 15px;
   line-height: 24px;
-  margin-bottom: 12px;
+  color: #344563;
 }
 
-nav a {
-  font-style: normal;
-  font-weight: bold;
-  font-size: 18px;
+.languages {
+  padding: 0 0 0 0;
+  display: flex;
+  align-items: center;
+  flex: start;
+}
+
+.language-button {
+  margin-right: 10px;
   color: #6070ff;
+  border: none;
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  font-weight: 500px;
+  font-size: 12px;
+  line-height: 16px;
+  padding: 4px 12px;
+  border-radius: 9px;
+  margin-bottom: 20px;
 }
 
-nav a,
-.headline a {
-  text-decoration: none;
+.languages li {
+  list-style: none;
+  margin-left: 0;
 }
 
-.headline a {
-  color: #7f8cff;
-  font-size: 16px;
+.see-project-button {
+  margin-bottom: 40px;
+}
+
+.see-project-button,
+.get-my-resume-button {
+  padding: 12px;
+  color: #396df2;
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  font-size: 17px;
   line-height: 24px;
+  border: 1px solid #396df2;
+  margin-bottom: 10px;
+  border-radius: 10px;
+  background-color: white;
+}
+
+.about-myself {
+  background-color: white;
+  display: flex;
+  flex-direction: column;
+  padding-left: 24px;
+  padding-right: 24px;
+  border-radius: 0 70px 0 0;
+  padding-top: 114px;
+  margin-bottom: 114px;
+}
+
+.about-myself h2 {
+  margin-top: 114px;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
   margin-bottom: 0;
 }
 
-.icon-container {
+.about-myself p {
+  margin-top: 12px;
+  margin-bottom: 12px;
+  font-family: 'poppins', Arial, Helvetica, sans-serif;
+  font-weight: 400px;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.about-myself .social-icons {
+  display: flex;
+  flex-direction: row;
   padding-left: 0;
-  padding-right: 24px;
-  margin-top: 0;
+  list-style: none;
+  margin-bottom: 22px;
+}
+
+.about-myself .icon-container {
+  margin-right: 22.92px;
+}
+
+.self-profession {
+  display: flex;
+  flex-direction: column;
+}
+
+ul .profesion-container {
+  display: flex;
+}
+
+.frameworks,
+.skills,
+.language-select {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.language-items {
+  display: flex;
+  flex-direction: column;
+  row-gap: 10px;
+}
+
+.dropdown {
+  width: 24px;
+  height: 40px;
+  padding-top: 25px;
+  padding-right: 42px;
```

Comment:
**[OPTIONAL] Kindly make sure you play with your  margins and paddings to reflect the Figma Design**

Your Design
![inentation](https://user-images.githubusercontent.com/26736582/198753310-2f023ca4-4b7f-455e-97e5-2182f32e87de.png)

Figma Design

![improve](https://user-images.githubusercontent.com/26736582/198753317-93322c76-bd4e-4162-96b3-89c4164619b1.png)